Jenkinsfile.update-api-sdk-php

```diff
@@ -0,0 +1,15 @@
+elifeUpdatePipeline(
+    { commit ->
+        lock('search--ci') {
+            builderDeployRevision 'search--ci', commit
+            builderCmd "search--ci", "composer update elife/api elife/api-client elife/api-sdk --no-interaction", "/srv/search"
```

Comment:
Might be best to go `elife/api elife/api-sdk --with-dependencies --no-suggest` (and treat `elife/api-client` as a dependency of `elife/api-sdk`).